pynsim/simulators/simulator.py

```diff
@@ -91,7 +91,21 @@ def __repr__(self):
         my_engines = ",".join([m.name for m in self.engines])
         return "Simulator(engines=[%s])" % (my_engines)
 
-    def start(self):
+    def initialise(self):
+        logging.info("Initialising simulation")
+        if self.network is None:
+            logging.critical("No network to simulate!")
+            return
+
+        if len(self.timesteps) == 0:
+            logging.critical("No timesteps specified!")
+            return
+
+        for engine in self.engines:
+            logging.debug("Setting up engine %s", engine.name)
+            engine.initialise()
```

Comment:
Is it intended that by setting initalise to false, none of the engines will be initialised?

Reply:
Incidenatlly, I have merged the previous MR into master, so the logging criticals are now runtime errors.

Reply:
The `initialise` keyword is only provided to the `start` method. It's meant to allow you to control the initialisation step separately to the running of the simulation step. If you've called `.initialise()` manually then you don't want to have that process repeated during `start()`.

Reply:
OK, understood. that's fine.